guides/common/modules/proc_creating-a-webhook-template.adoc

```diff
@@ -20,3 +20,30 @@ For more information, see {ManagingHostsDocURL}Template_Writing_Reference_managi
 . Optional: Enter the description and audit comment.
 . Assign organizations and locations.
 . Click *Submit*.
+
+.Examples
+When creating a webhook template, you must follow the format of the target application that the template is made for.
+For example, an application can expect a `"text"` field with the webhook message.
+Refer to the documentation of your target application to find more about how your webhook template format should look like.
+
+Example 1::
```

Comment:
```suggestion
Example "running remote execution jobs"::
```

Reply:
Since it is already under **Examples** I omitted the word "Example" from here altogether. Does that work?

Reply:
Even better!